src/components/ItaliaTheme/Blocks/RssBlock/CardWithImageRssTemplate.jsx

```diff
@@ -70,11 +70,9 @@ const CardWithImageRssTemplate = ({
                   )}
                   <CardBody tag="div" className="px-4">
                     <div className="category-top">
-                      {item?.categories?.length > 0 && item.categories[0]._ && (
+                      {item?.source?.length > 0 && (
```

Comment:
item.categories deve rimanere. 
In più, deve essere mostrata anche la sorgente.
Mostrerei prima la sorgente e poi le categorie.